### ISSUE TITLE
Feature Request, not a bug - love this plugin!

### ISSUE THREAD

**Issue by Zethin64** [McAJBen/animation-skipper#6]
Hullo!

It took me a little bitty to understand what the purpose of this plugin was, but as someone who runs multiple accounts I know see and really enjoy the benefit of this, a sort of blinder placed over the eyes of an account that is 'in-progress' with their afk action that they're currently farming... I love it! Saves me from wasting attention / energy and looking at that client to see if there's any changes that need to be made. Idle notifications work, but your brain notices better that the shaders are gone, as opposed to a notification or a color sitting on your screen.

My request, and feel free to shoot me down if this isn't possible. But one of my favorite AFK activities on my alt is Skeletal Wyverns, afking long combat that you can safe-spot. Is there, any way that safe-spot combat ranging/maging, could be added to the threshold of thingies that trigger the blinders to appear?

Regardless of your answer - thank you for this plugin! It's extremely useful, and well made. c:

**Comment by singatias** [comment]
I agree, I got confuse too and start  wondering if this wasn't cheating lol. Maybe update the description.
